### PR TITLE
Unfuckies pod blood

### PR DESCRIPTION
**Pull request by Time-Green**
I broke it in #72220
Thanks to @Fikou for catching this
list(variable = 0.1) doesnt work on byond, so I last-minute improvised and changed it to
list("[variable]" = 0.1), using a string instead of a typepath. I already tested it thoroughly so decided it was probably good without thinking of it anymore

:cl:
fix: fixes pod blood I swear
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/replicapod.dm [hunk 1]
```diff
@@ -205,10 +205,11 @@
 	podman.set_cloned_appearance()
 
 	//Get the most plentiful reagent, if there's none: get water
-	var/most_plentiful_reagent = list(/datum/reagent/water = 0)
+	var/list/most_plentiful_reagent = list(/datum/reagent/water = 0)
 	for(var/reagent in reagents_add)
 		if(reagents_add[reagent] > most_plentiful_reagent[most_plentiful_reagent[1]])
-			most_plentiful_reagent = list("[reagent]" = reagents_add[reagent])
+			most_plentiful_reagent.Cut()
+			most_plentiful_reagent[reagent] += reagents_add[reagent]
```
Comment (Fikou) [inline code/modules/hydroponics/grown/replicapod.dm:212]: you sure this shouldnt be just = ?

Comment (Time-Green) [reply code/modules/hydroponics/grown/replicapod.dm:212]: I don't think it really matters, I just used the same syntax as LAZYADDASSOC
(and yes it does work I tested the final product this time)

Comment (Time-Green) [reply code/modules/hydroponics/grown/replicapod.dm:212]: If you prefer I do it differently just tell me, I don't really care

Comment (Fikou) [reply code/modules/hydroponics/grown/replicapod.dm:212]: i think = is easier to understand

Comment (Time-Green) [reply code/modules/hydroponics/grown/replicapod.dm]: done

**Comment by YakumoChen** [comment]
Fixes #71636

**Review by tf-4 — APPROVED** [review]

**Comment by Time-Green** [comment]
> Fixes #71636 ??

Afraid not

**Pull request merged**